### ISSUE TITLE
Reporting student last activity reports the enrollment updated date instead of an actual date of last activity

### ISSUE THREAD

**Issue by thomasplevy** [gocodebox/lifterlms#332]
The issue lies in the file at https://github.com/gocodebox/lifterlms/blob/master/includes/admin/reporting/tables/llms.table.student.courses.php

The method:

https://github.com/gocodebox/lifterlms/blob/master/includes/admin/reporting/tables/llms.table.student.courses.php#L111-L113

Only returns the course enrollment updated date. We should be utilizing an `LLMS_Events_Query` instead.

**Comment by norgay-gmbh** [comment]
This issue is very important as we are offering courses with public certification and public funding. This bug is quite obvious & critical and is undermining the credibility of all data displayed in LIfterLMS.

**Comment by brianhogg** [comment]
The label “Last Activity Date” is incorrect, as it always gets the enrollment updated date. Calling that “Enrollment Updated Date” or something would be more accurate.

Most events at the course level are not tracked as events, apart from video watching on lessons when using Advanced Videos. So we’d need to piece things together from data elsewhere, or start tracking more events in the events table. Things like re-marking a lesson as complete may not update the date again. I don’t think we’d want to get to the google analytics level and start tracking pageviews on lessons necessarily due to the load it would add on to servers.

In summary it’s not a quick or easy change to accurately show “last activity date” at the course level for a student and have it be at all accurate. We've added this on the list as a potential enhancement to spend more time on planning and figuring out a way to have “Last activity date” be the date the user last did something within a course. In the meantime #2898 will rename the label.
